### PR TITLE
Update Oracle JDK 8 to version 8u31

### PR DESCRIPTION
**Pull request by GeBeater**

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -124,12 +124,12 @@
...
 # x86_64
-default['java']['jdk']['8']['x86_64']['url'] = 'http://download.oracle.com/otn-pub/java/jdk/8u25-b17/jdk-8u25-linux-x64.tar.gz'
-default['java']['jdk']['8']['x86_64']['checksum'] = 'e145c03a7edc845215092786bcfba77e'
+default['java']['jdk']['8']['x86_64']['url'] = 'http://download.oracle.com/otn-pub/java/jdk/8u31-b13/jdk-8u31-linux-x64.tar.gz'
+default['java']['jdk']['8']['x86_64']['checksum'] = '173e24bc2d5d5ca3469b8e34864a80da'
 
 # i586
-default['java']['jdk']['8']['i586']['url'] = 'http://download.oracle.com/otn-pub/java/jdk/8u25-b17/jdk-8u25-linux-i586.tar.gz'
-default['java']['jdk']['8']['i586']['checksum'] = 'b5b16247f66643727d9b6d4bc7c5efda'
+default['java']['jdk']['8']['i586']['url'] = 'http://download.oracle.com/otn-pub/java/jdk/8u31-b13/jdk-8u31-linux-x64.tar.gz'
```
Comment (erichelgeson) [inline attributes/default.rb]: You have the wrong filename here.

File: attributes/default.rb [hunk 2]
```diff
@@ -124,12 +124,12 @@
 # Official checksums for the latest release can be found at https://www.oracle.com/webfolder/s/digest/8u25checksum.html
```
Comment (erichelgeson) [inline attributes/default.rb]: Can you update the url here?

**Comment by erichelgeson** [comment]
Thanks for the contribution, few updates and i'll merge.

File: attributes/default.rb (hunk 2 above)
Comment (GeBeater) [inline attributes/default.rb]: Done! ;-)

**Comment by erichelgeson** [comment]
Merged in here d3407139d822e623270962b6022f0185b30a9710 thanks!

**Pull request closed without merging**